### DISCUSSION TITLE
start.spring.io support

### DISCUSSION THREAD

**Discussion by anbusampath** [awspring/spring-cloud-aws#423] [General]
I saw [comment](https://github.com/spring-io/start.spring.io/pull/868#issuecomment-1064923304) in start.spring.io issue that spring-cloud-aws removed. Is it possible to add the support back since 3.0.0 M1 getting out.

**Comment by maciejwalkowiak** [comment]
Thanks for reaching out @anbusampath! We do consider getting back to start.spring.io but only after 3.0 is released and we also receive proper funding for the project.

**Reply by anbusampath** [reply]
Thanks @maciejwalkowiak
